src/i18n.js

```diff
@@ -9,7 +9,7 @@ i18n
     lng: sessionStorage.getItem('rsdk_locale')?.substring(0, 2) || 'en',
     backend: {
       /* translation file path */
-      loadPath: `${window.location.pathname}assets/i18n/{{lng}}.json`
+      loadPath: `${window.location.pathname.replace('/cookies','')}assets/i18n/{{lng}}.json`
```

Comment:
Do we need this script, since you have moved the init to appSelector ?

Reply:
No I guess not - refactored here [e312177](https://github.com/norm-l/odx/pull/123/commits/e312177a419463cdc4dbb3759f01a8bebda331e7)